docs/recipes/contribute-recipes.md

```diff
@@ -0,0 +1,69 @@
+
+A tutorial, on the other hand, is Julia Child holding your hand through the entire cooking process: explaining each ingredient and procedure, demonstrating best practices, pointing out potential problems, and generally making sure you can’t stray from the happy path to a delicious meal.
+
+We love Julia, and we love tutorials. But we don’t expect that a Prefect recipe should handhold users through every step and possible contingency of a solution. A recipe can start from an expectation of more expertise and problem-solving ability on the part of the reader.
+
+To see an example of a high quality recipe, check out **[Serverless with AWS Chalice](https://github.com/PrefectHQ/prefect-recipes/tree/main/flows-advanced/serverless)**. This recipe includes all of the elements we like to see.
+
+## Steps to adding your recipe
+
+Here’s our guide to creating a recipe:
+
+1. Clone the [Prefect Recipes repo](https://github.com/PrefectHQ/prefect-recipes) and create a branch.
+2. [Add your code](#what-are-the-common-ingredients-of-a-good-recipe). The code may simply be a copy/paste of a personal project, whether that be a single Python file or an entire folder/repo in itself. Unsure of where to add your code? Just add it to the `flows-advanced/` folder. A Prefect maintainer will help you find a better place for the recipe if there is one.
+3. (Optional) Write a [README](#what-are-some-tips-for-a-good-recipe-readme).
+4. Include a dependencies file, if applicable.
+6. Push your code and make a PR to the repo.
+
+That’s really it! 
+
+## What makes a good recipe?
+
+Every recipe is useful, as other Prefect users can adapt the recipe to their needs. Particularly good ones help a Prefect user bake a great dataflow solution! Take a look at the [prefect-recipes repo](https://github.com/PrefectHQ/prefect-recipes) to see some examples.
+
+## What are the common ingredients of a good recipe?
+
+- Easy to understand: Can a user easily follow your recipe? Would a README or code comments help? A simple explanation providing context on how to use the example code is useful, but not required. A good README can set a recipe apart, so we have some additional suggestions for README files below.
+- Code and more: Sometimes a use case is best represented in Python code or shell scripts. Sometimes a configuration file is the most important artifact &mdash; think of a Dockerfile or Terraform file for configuring infrastructure.
+- All-inclusive: Share as much code as you can. Even boilerplate code like Dockerfiles or Terraform or Helm files are useful. Just *don’t share company secrets or IP*.
+- Specific: Don't worry about generalizing your code: Aside from removing anything internal/secret, don’t worry about it! Other users will extrapolate their own unique solutions from your example.
```

Comment:
```suggestion
- Specific: Don't worry about generalizing your code, aside from removing anything internal/secret! Other users will extrapolate their own unique solutions from your example.
```

I think we can have "don't worry" either at the beginning or at the end of the sentence but preferably not both.